### PR TITLE
one unique 'Orders' management view & refund to front-end for past order

### PR DESCRIPTION
**Pull request by caburj**
[IMP] point_of_sale,pos_*: combine ticket and management screens

We are referring to TicketScreen and OrderManagementScreen.
The two screens almost serves the same purpose. There is not much reason
to keep them separate. We combine the two in this commit. Basically,
we remove OrderManagementScreen and its components. Some are kept and
moved to the TicketScreen.

TASK-ID: 2481404

---

[IMP] point_of_sale,pos_*: refund orders thru POS UI

* Perform refunds in the POS UI.
* Select orderlines and quantity to refund.
* Add refund orderlines in the active order.

TASK-ID: 2481561

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/models/pos_order.py [hunk 1]
```diff
@@ -287,6 +287,25 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
...
+
+    @api.depends('lines.refund_orderline_ids', 'lines.refunded_orderline_id')
+    def _compute_refund_related_fields(self):
+        for order in self:
+            order.refund_orders_count = len(order.mapped('lines.refund_orderline_ids.order_id'))
+            order.is_refunded = order.refund_orders_count > 0
+            order.refunded_order_ids = order.mapped('lines.refunded_orderline_id.order_id')
+            order.refunded_orders_count = len(order.refunded_order_ids)
+
+    api.depends('lines.refunded_qty', 'lines.qty')
```
Comment (pimodoo) [inline addons/point_of_sale/models/pos_order.py:304]: missing "@" (@api)

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

File: addons/point_of_sale/models/pos_order.py [hunk 2]
```diff
@@ -396,7 +415,7 @@ def create(self, values):
     @api.model
     def _complete_values_from_session(self, session, values):
         if values.get('state') and values['state'] == 'paid':
-            values['name'] = session.config_id.sequence_id._next()
+            values['name'] = self._compute_order_name()
```
Comment (jerzyk) [inline addons/point_of_sale/models/pos_order.py:418]: this is wrong, new method `_compute_order_name` expect instance of the the model not recordset but this method (`_complete_values_from_session`) is called from recordset, so self has no values or different values

**Pull request closed without merging**